worlds/checksfinder/__init__.py

```diff
@@ -15,7 +15,7 @@ class ChecksFinderWeb(WebWorld):
         "single-player, multiworld, and related software.",
         "English",
         "checksfinder_en.md",
```

Comment:
You need to also change this to setup_en.md and rename the file.

> The file name you use aren't particularly important, but it should be descriptive of what
the tutorial is covering, and the name of the file must match the relative URL provided in the `Tutorial`. Currently,
the JS that determines this ignores the provided file name and will search for `game/document_lang.md`, where
`game/document/lang` is the provided URL.

https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md#tutorials

Reply:
okay go fix